ScrumPoker.Data/Data/GameRoomDataBase.cs

```diff
@@ -0,0 +1,47 @@
+using ScrumPoker.Core.Models;
+using ScrumPoker.DataBase.Interfaces;
+
+namespace ScrumPoker.Data.Data;
+
+public class GameRoomDataBase : IGameRoomData
```

Comment:
If interface has only 1 implementation then class name + interface name should be like:
`MyClass` and `IMyClass`
Also for now rename this to `GameRoomRepository`